specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3562,6 +3562,462 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence": {
+      "put": {
```

Comment:
This API is for creating new Indicator. We cant follow the requirement PUT {resourceType}/{resourceName} because:
Our resource, which is indicators, does not have an unique name. The main unique field in an indicator is ID. ID is generated by the back-end and used to support partitioning in database. Our design does not allow customers to set the id.

Reply:
The current shape of the API amounts to doing a PUT on a collection, which is not allowed.

---
In reply to: [481459273](https://github.com/Azure/azure-rest-api-specs/pull/10463#discussion_r481459273) [](ancestors = 481459273)